### ISSUE TITLE
[release]: Signed Tableau Connector for Certification

### ISSUE THREAD

**Issue by GumpacG** [opensearch-project/opensearch-build#3631]
### Did you read the on-boarding document

Yes

### What is the name of your component?

signed JDBC Tableau Connector

### What is the link to your GitHub repo?

https://github.com/opensearch-project/sql-jdbc

### Targeted release date

ASAP in order to submit for Tableau certification

### Where should we publish this component?

artifacts: https://artifacts.opensearch.org/

### What type of artifact(s) will be generated for this component?

Tableau Connector

### Have you completed the required reviews including security reviews, UX reviews?

Yes

### Have you on-boarded automated security scanning for the GitHub repo associated with this component?

Yes

### Additional context

A signed Tableau Connector is required for submissions to be one of Tableau's certified connectors.
The connector can be found in https://github.com/opensearch-project/sql-jdbc/actions/runs/5282016841
Please rename the file to have the version in the filename. For example, `opensearch_sql_jdbc-v1.0.taco`

**Comment by Yury-Fridlyand** [comment]
Depends on #3614

**Comment by rishabh6788** [comment]
Signed artifact has already been uploaded to artifacts bucket, see #3614

**Comment by rishabh6788** [comment]
@GumpacG I downloaded taco file from https://github.com/opensearch-project/sql-jdbc/actions/runs/5282016841 and the file name is opensearch_sql_jdbc.taco 2. Should I rename it to openserach_sql_jdbc-v1.0.taco sign and upload?

**Comment by rishabh6788** [comment]
Signed `taco` file uploaded and can be downloaded from https://artifacts.opensearch.org/opensearch-clients/jdbc/opensearch_sql_jdbc-v1.0.taco. 
Please note we use centralized signing infra and all the files are signed by `Amazon Web Services, Inc.`
@GumpacG

**Comment by GumpacG** [comment]
Thank you @rishabh6788

**Comment by GumpacG** [comment]
Reopening this issue as an updated connector needs to be resigned for Tableau submission.

The connector can be downloaded from https://github.com/opensearch-project/sql-jdbc/actions/runs/5695042862.
Please rename the file to have the version in the filename. For example, `openserach_sql_jdbc-v1.0.taco`.
Request is for the signed connector to be downloadable from the same url (https://artifacts.opensearch.org/opensearch-clients/jdbc/opensearch_sql_jdbc-v1.0.taco).

**Comment by GumpacG** [comment]
Closing issue to create a new issue with a different version number.
